index.html

```diff
@@ -9256,6 +9256,7 @@ <h5>Presentational Role Inheritance</h5>
 			<rdef>textbox</rdef>
 			<div class="role-description">
 				<p>A type of input that allows free-form text as its value.</p>
+				<p>Authors MUST limit its children to completely presentational elements (like a spacer image, decorative graphic, or clearing element).</p>
```

Comment:
do we need the parenthetical examples?  

if examples are needed, maybe they should be more specific to what someone might visually display in a textbox - such as a success or error icon that visually communicates the field's `aria-invalid` state?  these could replace the references to a "spacer image" and "decorative graphic".   Those two examples should be enough, if needed, and then the "clearing element" can be dropped all together. I'm at a loss for what this means outside of a "clear button", and that would be the opposite of the guidance this PR is adding.